PIL/Image.py

```diff
@@ -2142,6 +2237,7 @@ def __transformer(self, box, image, method, data,
         self.im.transform2(box, image.im, method, data, resample, fill)
```

Comment:
Probably. It's technically accessible part of the public interface, but as part of the core object it's something that we can change reasonably carefully. 

I'm kind of wondering what became of `im.transform`, since we're just left with `im.transform2`